stable/vulcan/values.yaml

```diff
@@ -376,11 +376,9 @@ api:
     bucketLogs: logs
     s3PrivateBucket: insights
     s3PublicBucket: public-insights
-    queueArn: arn:aws:sqs:local:012345678900:VulcanK8SAPIScans
```

Comment:
Remove VulcanK8SAPIScans from goaws.config.goaws.yaml in values.yaml

Reply:
👍

Reply:
You removed the subscription, but the queue is still there

Reply:
True!!, it's still there, 🤔 I think I lost the removal when performed the rebase thanks again!!